src/css/main.less

```diff
@@ -133,6 +133,10 @@ a,
   margin-bottom: 0;
 }
 
+.btn.pull-right {
+  margin-left: 5px;
```

Comment:
So the buttons should have a gap between each other, if you have more then one..
https://cloud.githubusercontent.com/assets/859154/9138795/679474ae-3d26-11e5-8fac-cc06d71b722b.png
I think this is not comment-worth.